src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,54 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types';
+
+type Props = {
+  tabs: Tab[],
+  selectedTab: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTab,
+  onTabSelected,
+}) => {
+  const hanldeTabSelected = (tab: Tab) => {
+    if (tab.id !== selectedTab) {
```

Comment:
So here is a problem. Your `selectedTab` is whole tab object. But you compare id and an object. It isn't work correct